inference-engine/src/transformations/include/transformations/swish_fusion.hpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <utility>
+
+#include <transformations_visibility.hpp>
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API SwishFusion;
+class TRANSFORMATIONS_API SwishFusionWithSigmoid;
+class TRANSFORMATIONS_API SwishFusionWithSigmoidWithBeta;
+class TRANSFORMATIONS_API SwishFusionWithBeta;
+class TRANSFORMATIONS_API SwishFusionWithoutBeta;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief SwishFusion transformation replaces various sub-graphs to Swish op.
+ */
+class ngraph::pass::SwishFusion: public ngraph::pass::GraphRewrite {
+public:
+    SwishFusion() {
+        add_matcher<ngraph::pass::SwishFusionWithSigmoid>();
+        add_matcher<ngraph::pass::SwishFusionWithSigmoidWithBeta>();
+        add_matcher<ngraph::pass::SwishFusionWithBeta>();
+        add_matcher<ngraph::pass::SwishFusionWithoutBeta>();
+    }
+};
+
+class ngraph::pass::SwishFusionWithSigmoid: public ngraph::pass::MatcherPass {
```

Comment:
Please add description for every MatcherPass transformations

Reply:
Done